lambda/service/go.mod

```diff
@@ -2,4 +2,27 @@ module github.com/pennsieve/integration-service/service
 
 go 1.20
 
-require github.com/aws/aws-lambda-go v1.41.0
+require (
+	github.com/aws/aws-lambda-go v1.41.0
+	github.com/pennsieve/pennsieve-go-core v1.8.0
+)
+
+require (
+	github.com/aws/aws-sdk-go-v2 v1.17.8 // indirect
+	github.com/aws/aws-sdk-go-v2/config v1.18.14 // indirect
+	github.com/aws/aws-sdk-go-v2/credentials v1.13.14 // indirect
+	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.12.23 // indirect
+	github.com/aws/aws-sdk-go-v2/feature/rds/auth v1.2.7 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/configsources v1.1.32 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.4.26 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/ini v1.3.30 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.9.23 // indirect
+	github.com/aws/aws-sdk-go-v2/service/sso v1.12.3 // indirect
+	github.com/aws/aws-sdk-go-v2/service/ssooidc v1.14.3 // indirect
+	github.com/aws/aws-sdk-go-v2/service/sts v1.18.4 // indirect
+	github.com/aws/smithy-go v1.13.5 // indirect
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/lib/pq v1.10.7 // indirect
+	github.com/sirupsen/logrus v1.9.0 // indirect
+	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8 // indirect
+)
```

Comment:
@jwagenaar adding go-core results in a lot of dependencies. Perhaps we can think around whether all these are indeed required for what we need to accomplish, or whether just what is required can be implemented within the service itself.

Reply:
I think it is fine to explore, but want to stay very far away from duplicating code across multiple repos / services.